javascript/react/introduction-to-hooks/rules-of-hooks.md

```diff
@@ -0,0 +1,104 @@
+---
+author: kapnobatai136
+
+levels:
```

Comment:
FYI `levels` are fully deprecated, there should be no need to use them anymore.

CC @mihaiberq for confirmation

Reply:
Yup, you can omit them

Reply:
removed all the `levels` from the insights